cmd/config.go

```diff
@@ -0,0 +1,68 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v3"
+)
+
+var (
+	DefaultHost   = "localhost"
+	FileName      = ".guardian"
+	FileExtension = "yaml"
+)
+
+type config struct {
+	Host string `yaml:"host"`
```

Comment:
what's the difference between this config and app.Config?
does it make sense to merge them?

Reply:
I was also having the same doubt, but the difference for cmd.config is that it is only the config for the CLI to run, while app.Config is more into the server's config